app/models/storage.rb

```diff
@@ -77,6 +78,10 @@ def ext_management_systems
       :host_storages => {:storage_id => id}).distinct.to_a
   end
 
+  def storage_clusters
+    parents().select{ |parent| parent.kind_of?(StorageCluster) }
```

Comment:
Robocop also flagged me. We'll have that plus a couple style fixes ready in next push